### PR TITLE
Add more information about releasing

### PR DESCRIPTION
**Pull request by dfithian**

### CODE REVIEW

File: scripts/hackage-docs.sh [hunk 1]
```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+set -e
+
+# This if stack-enabled fork of https://github.com/ekmett/lens/blob/master/scripts/hackage-docs.sh
```
Comment (mossprescott) [inline scripts/hackage-docs.sh]: "This if fork"?

**Comment by mossprescott** [comment]
If you re-publish to get the new info into hackage, do you have to bump the version, or can you replace the one that's there now? I guess that would be fine since the code hasn't changed.

File: RELEASING.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+Outline for releasing this package to Hackage and Stackage
+
+## Steps
+
+* `git tag <version> -m "Release version <version>"`
+* `git push origin --tags`
+* `stack sdist`
+* Copy to a folder that can be navigated to for upload
+* Upload here: http://hackage.haskell.org/packages/upload
+* `scripts/hackage-docs.sh <user>`
```
Comment (DanBurton) [inline RELEASING.md]: What's the point of using hackage-docs.sh. Isn't the hackage automatic doc generation sufficient for our needs?

**Comment by dfithian** [comment]
I have to upload version 0.1.1, so I will bump the version.

File: RELEASING.md (hunk 2 above)
Comment (dfithian) [reply RELEASING.md]: I haven't ever gotten that to work. This is what I ran to get the docs up in their current state, this stuff is just fire and forget for me.

Comment (DanBurton) [reply RELEASING.md]: Cool. As long as this works, I have no objections.

**Pull request merged**